src/main/java/com/terraformersmc/campanion/block/RopeBridgePostBlock.java

```diff
@@ -60,11 +60,13 @@ public ActionResult onUse(BlockState state, World world, BlockPos pos, PlayerEnt
                             int itemsToUse = Math.round(xzDist / RopeBridge.BLOCKS_PER_ROPE);
                             if(stack.getCount() < itemsToUse) {
                                 reason = Optional.of(new TranslatableText("message.campanion.rope_bridge.ropes", itemsToUse - stack.getCount()));
+                            } else {
+                                stack.decrement(itemsToUse);
                             }
-                            stack.decrement(itemsToUse);
                         }
                         if(reason.isPresent()) {
                             player.addChatMessage(reason.get(), false);
+                            return ActionResult.SUCCESS;
```

Comment:
I wasn't sure whether to return `SUCCESS` or `FAIL` here, but since line 75 returns `SUCCESS` for an error I figured I'd do the same